### DISCUSSION TITLE
Dag with cron "0 11 1 * *" to execute job monthly at 11 am is not being triggered

### DISCUSSION THREAD

**Discussion by akanshajainnice** [apache/airflow#21791] [Q&A] [answered]
One of the dags having monthly cron set as "0 11 1 * *" but it does not trigger as per scheduled interval.  The dag was created in 06/12/ 2021. With cron schedule, the job was expected to trigger automatically on 1st Jan 2022 and 1st Feb 2022. But it did not trigger on time.
Airflow version : 1.10.10
default_args = {
    'owner': 'airflow',
    'depends_on_past': False,
    'start_date': airflow.utils.dates.days_ago(1),
    'email_on_failure': False,
    'email_on_retry': False,
    'retries': 1,
    'retry_delay': timedelta(minutes=5),
}
Can you help me understand. why?

**Comment by potiuk** [comment, marked as answer]
Use fixed date as "start_date" not days_ago. Days ago is bad when used in this context and we are deprecating the use of it (all our examples are already converted to not use it).
